tutorials/tree/dictionary/README.md

```diff
@@ -0,0 +1,42 @@
+How to read and write custom classes in TTrees
+==============================================
+
+# Starting steps
+
+1. Build the project, it requires ROOT
+
+```
+cmake -B build -S . 
+cmake --build build 
+```
+
+2. Run the executable
+
+```
+./build/treeExample
+```
+
+3. Inspect the output rootfile, and the source code. 
+
```

Comment:
In general, let's also briefly describe here the steps required to read/write user-defined classes, i.e. why ROOT relies on dictionaries for class introspection, how to generate one, etc.